subsys/bluetooth/host/Kconfig

```diff
@@ -129,6 +129,35 @@ config BT_DRIVER_RX_HIGH_PRIO
 	int
 	default 6
 
+config BT_CONN_TX_WQ
+	bool "Use a separate workqueue for connection TX notify processing"
+	depends on BT_CONN_TX
+	help
+	  Use a separate workqueue instead of system workqueue for
+	  bt_conn_tx_notify processing. The option can be used to make Bluetooth
+	  stack more independent from the system workqueue.
+
+if BT_CONN_TX_WQ
+
+config BT_CONN_TX_WQ_STACK_SIZE
+	int "Stack size of workqueue for connection TX notify processing"
+	default 1200
+
+config BT_CONN_TX_WQ_PRIO
+	# Hidden option for cooperative connection TX notify workqueue priority
+	int
+	default 8
```

Comment:
I did it in the same way as for `CONFIG_BT_RX_PRIO` and `CONFIG_BT_DRIVER_RX_HIGH_PRIO`. Still, it would not hurt to make the priority user-configurable. Updated.

Reply:
Yeah, checked those out too, and didn't really understand why they were non-configurable Kconfig options either :)

Reply:
I assumed that it's done to prevent the user from breaking something while still allowing to override the value by redefining the default in application's Kconfig (that's why I initially followed the same approach here)

Reply:
That pretty much sounds like "We want it to be configurable but not really" :D Or "Configurable but with extra steps". 

`int` Kconfigs are generally just a pain to work with, as they are not as flexible as `bool`s (`imply` `select` only works with `bool`s).